docs/pages/deploy/build-project.mdx

```diff
@@ -1,125 +1,30 @@
-
-## Developer account
-
-You will need to have a developer account for the app store you want to submit your app.
-
-<Collapsible summary="Google Play Developer membership is required to distribute to the Google Play Store.">
-
-You can build and sign your app using EAS Build, but you can't upload it to the Google Play Store unless you have a membership, a one-time $25 USD fee.
-
-</Collapsible>
-
-<Collapsible summary="Apple Developer Program membership is required to build for the Apple App Store.">
-
-If you are going to use EAS Build to create release builds for the Apple App Store, you need access to an account with a $99 USD [Apple Developer Program](https://developer.apple.com/programs) membership.
-
-</Collapsible>
-
-## App signing credentials
-
-Before the build process can start for app stores, you will need to have a store developer account and generate or provide app signing credentials.
-
-Whether you have experience with generating app signing credentials or not, EAS CLI does the heavy lifting. You can opt-in for EAS CLI to handle the app signing credentials process.
-
-### Android app signing credentials
-
-- If you have not yet generated a keystore for your app, you can let EAS CLI take care of that for you by selecting `Generate new keystore`, and then you are done. The keystore is stored securely on EAS servers.
-- If you want to manually generate your keystore, see the [manual Android credentials guide](/app-signing/local-credentials#android-credentials) for more information.
-
-### iOS app signing credentials
-
-- If you have not generated a provisioning profile and/or distribution certificate yet, you can let EAS CLI take care of that for you by signing in to your Apple Developer Program account and following the prompts.
-- If you want to manually generate your credentials, see the [manual iOS credentials guide](/app-signing/local-credentials#ios-credentials) for more information.
-
-## Wait for the build to complete
 
-By default, the `eas build` command will wait for your build to complete, but you can interrupt it if you prefer not to wait. Monitor the progress and read the logs by following the link to the build details page that EAS CLI prompts once the build process gets started. You can also find this page by visiting [your build dashboard](https://expo.dev/builds) or running the following command:
+To submit your app to the app stores, you'll need to create a production build. This build is optimized for performance and size, and it includes all the necessary assets and configurations for the app stores.
 
-<Terminal cmd={['$ eas build:list']} />
+## Prerequisites
 
-If you are a member of an organization and your build is on its behalf, you will find the build details on [the build dashboard for that account](https://expo.dev/accounts/[account]/builds).
+Paid developer accounts are required to submit builds to the app stores. You will need the following:
 
-## Production builds locally
```

Comment:
I'm confused about what you're asking. --I don't want to create more docs about building locally, unless you think we need them.

Context: I chose to link to the `eas build --local` doc because I thought it was the same as doing the manual steps on reactnative.dev, just automated. I may be wrong about that! I'm not too familiar.

For this PR, do you think I should:
1. Remove this section entirely?
2. Link to the React Native docs again and remove the /local-builds link?

Reply:
Apologies for leaving an open-ended question. To give some context to this section. There was some strong user feedback before about not being able to create a production release build locally. However, I understand that the premise of these sections in docs Home is meant to relate directly to EAS. The instruction to create a local production release is different from React Native docs when using prebuild (local app compilation) with Expo CLI. 

I've talked with Brent a little bit about this and have created a follow up linear task to talk about local app compilation in relation to production release separately: https://linear.app/expo/issue/ENG-12981/create-a-page-to-provide-steps-on-how-to-create-a-production-build. 

We should not block this PR from releasing and I will follow up on the linear task linked above next week.